staging/src/k8s.io/apiserver/pkg/cel/common/values_test.go

```diff
@@ -318,6 +318,16 @@ func TestToValue(t *testing.T) {
 				"s2": struct2,
 			},
 		},
+		{
+			name:       "compare: identical complex structs",
+			expression: "c1 == c2",
+			activation: map[string]typedValue{"c1": complex1, "c2": complex1Again},
+		},
```

Comment:
This failed on `nilBytes` field when the unstructured representation contained a map with `nilBytes: nil`.